berate_ap

```diff
@@ -1860,6 +1873,12 @@ ht_capab=${HT_CAPAB}
 EOF
 fi
 
+if [[ $IEEE80211W -eq 1 ]]; then
```

Comment:
What does hostap do by default? you set this to 2 earlier and I dont see it being used anywhere. What is the 2 for?